README.md

```diff
@@ -1,19 +1,43 @@
-# IODigital_CriticalCss
-This node module makes it possible to generate critical css bundles from different pages in different viewports. It is based on [GitHub - addyosmani/critical: Extract & Inline Critical-path CSS in HTML pages](https://github.com/addyosmani/critical)
+# @iodigital/criticalcss
+
+This node module makes it possible to generate critical css bundles from different pages in different viewports. It uses [GitHub - addyosmani/critical](https://github.com/addyosmani/critical) under the hood.
 
-## Install
-`npm install iodigital-criticalcss`
 
 ## Usage
 
+### Install
+```sh
+npm install @iodigital/criticalcss
+```
+
 ### Configuration
-First add a configuration file in the root of your project , named: `criticalcss.config.json` .
-This configuration should contain your urls and dimensions for the viewport. Example configuration:
+First add a configuration file in the root of your project, named `criticalcss.config.js`. This file should contain a `const config` for generating critical css and a `const pages` with the urls and dimensions for the viewport. Example configuration:
 
-```
-{
+```js
+const config = {
+    rebase: (asset) => { // absolute assets path
+        if (asset.pathname !== null && asset.pathname === asset.originUrl) {
+            return asset.absolutePath;
+        }
+        return asset.url;
+    },
+    dimensions: [
+        {"width": 320}, // height is optional
+        {"width": 767, "height": 3000},
+        {"width": 992},
+        {"width": 1200},
+        {"width": 1424}
+    ],
+    penthouse: {
+        timeout: 60000,
+        forceExclude: [
+            ".footer__columns--wrapper>.footer__column:nth-of-type(4) img"
+        ]
+    }
+}
+
+const pages = {
     "domain": "https://site.com",
```

Comment:
Dit is helemaal buiten de scope van deze MR, maar zo te zien is er nu maar één domain mogelijk maar wel meerdere themes afhankelijk van url prefixes, maar Magento staat het wel toe om meerdere domains te kunnen gebruiken binnen een installatie.
Je zou van pages een array kunnen maken met dit type object zodat je de optie hebt om meerdere domains met bijbehorende pages toe te voegen.

Reply:
Moved to new issue